content/src/content/jcr_root/apps/core/wcm/components/tabs/v1/tabs/_cq_dialog/.content.xml

```diff
@@ -72,6 +72,15 @@
                                         jcr:primaryType="nt:unstructured"
                                         sling:resourceType="granite/ui/components/coral/foundation/container">
                                         <items jcr:primaryType="nt:unstructured">
+                                            <label
```

Comment:
Same comment as in the Navigation component, this should be the last field in the dialog or in its own accessibility tab.

Reply:
Moved to a separate accessibility tab